product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx

```diff
@@ -100,7 +100,7 @@ Provision your cluster before configuring IAM for Postgres.
     1. On the **Additional Settings** tab, under **Authentication**, select **Identity and Access Management (IAM) Authentication**. 
     1. Select **Create Cluster** or **Save**.
     !!!note
-    To turn on IAM authentication using the CLI, see the [Using IAM authentication on AWS](/biganimal/latest/reference/cli/#using-iam-authentication-on-aws) section in the Using the BigAnimal CLI topic.
+    To turn on IAM authentication using the CLI, see the [Using IAM authentication on AWS](/biganimal/latest/reference/cli/using_features/#iam-authentication-cli-commands) section in the Using the BigAnimal CLI topic.
```

Comment:
Loving the loop - that links to a short section that ends with "Find out more about IAM for Postgres" and links back here.

Reply:
I'm open to suggestions. That link goes to the section we have in our measly CLI topics and is specific about the CLI commands. I saw no harm in linking to our measly IAM section in the _01postgres_access topic, which is all we have in terms of our IAM support at the moment. The IAM support itself currently is fairly measly too. My thinking was if a user started with the CLI topic, we should provide a link for more info, but maybe that is not useful. BTW, I use that same model for many of the CLI sections, where I link back to topics that have more information about the feature.

Reply:
I don't think it's a problem for us. This is just an edge case where the link back is so close to the landing site.